modules/carto/src/api/maps-v3-client.js

```diff
@@ -67,19 +67,33 @@ function dealWithError({response, error}) {
 /**
  * Build a URL with all required parameters
  */
-function getParameters({type, source}) {
+function getParameters({type, source, geoColumn, columns}) {
   const encodedClient = encodeParameter('client', 'deck-gl-carto');
   const parameters = [encodedClient];
 
   const sourceName = type === MAP_TYPES.QUERY ? 'q' : 'name';
   parameters.push(encodeParameter(sourceName, source));
 
+  if (geoColumn) {
```

Comment:
This parameter is only available for MAP_TYPES.TABLE, shouldn't be passed in other cases